docs/topics/booleans.md

```diff
@@ -1,6 +1,10 @@
 [//]: # (title: Booleans)
 
-The type `Boolean` represents boolean objects that can have two values: `true` and `false`.
+The type `Boolean` represents boolean objects that can have two values: `true` and `false`. 
+
+> On the JVM, the primitive `boolean` type typically uses 8 bits.
```

Comment:
I noticed that we don't mention primitives before this. Can you introduce the `boolean` type before this? Or do we know the size of `Boolean` objects that we can use instead?